app/controllers/bridges/oai/oai_controller.rb

```diff
@@ -172,6 +179,28 @@ def export_record(xml, task, format)
         end
       end
 
+      def export_resumption_token(xml, records, complete_list_size)
+        returned_records = records[...MAX_RECORDS_PER_RESPONSE]
+        cursor = @resumption_params.present? ? @resumption_params[:cursor] : 0
+
+        if records.size > MAX_RECORDS_PER_RESPONSE
+
+          new_token = @oai_params.merge(
+            {
+              last_id: returned_records.last.id,
+              ts_from: returned_records.last.updated_at.strftime('%Y-%m-%dT%H:%M:%S.%NZ'),
+              ts_until: @resumption_params.present? ? @resumption_params[:ts_until] : Time.zone.now.strftime('%Y-%m-%dT%H:%M:%S.%NZ'),
+              cursor: cursor + returned_records.size,
+            }
+          )
+
+          xml.resumptionToken Base64.encode64(new_token.to_json), cursor:, completeListSize: complete_list_size
```

Comment:
Currently, this encoding adds line breaks. We can avoid so when using another method:

```suggestion
          xml.resumptionToken Base64.urlsafe_encode64(new_token.to_json), cursor:, completeListSize: complete_list_size
```

Reply:
That is a good catch. I changed that but since this had to be changed in the tests as well I created an own commit. Thanks!